packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -85,6 +85,18 @@ const ChartsTooltipRoot = styled(Popper, {
   zIndex: theme.zIndex.modal,
 }));
 
+/**
+ * Demos:
+ *
+ * - [Bars demonstration](https://mui.com/x/react-charts/bar-demo/)
+ * - [Lines demonstration](https://mui.com/x/react-charts/line-demo/)
+ * - [Area demonstration](https://mui.com/x/react-charts/area-demo/)
+ * - [Pie demonstration](https://mui.com/x/react-charts/pie-demo/)
```

Comment:
```suggestion
 * - [Bars demonstration](https://mui.com/x/react-charts/bars/)
 * - [Lines demonstration](https://mui.com/x/react-charts/lines/)
 * - [Pie demonstration](https://mui.com/x/react-charts/pie/)
```

Reply:
I tend to think it's better to link people chare their is some explanation. But might be wrong